specs/0119-gamification-badges.md

```diff
@@ -0,0 +1,84 @@
+
+### Translation Champion Badge
+**Description:** Rewards contributors for their efforts in submitting translations.
+
+**Criteria:**
+- **Translator Level 1:** Submit 5 translations.
+- **Translator Level 2:** Submit 50 translations.
+- **Translator Level 3:** Submit 250 translations.
+- **Translator Level 4:** Submit 1000 translations.
+
+### Community Builder Badge
+**Description:** Recognizes Locale Managers who successfully promote other contributors to new roles (contributor → translator and/or translator → team manager), fostering growth and leadership within the community.
+
+**Criteria:**
+- **Mentor Level 1:** Successfully promote 1 contributor to a new role.
+- **Mentor Level 2:** Successfully promote 2 contributors to a new role.
+- **Mentor Level 3:** Successfully promote 5 contributors to a new role.
+
+### Status Icons
+**Description:** Visual indicators awarded based on user roles and contributions, providing immediate context to other community members.
+
+**Icons:**
+- **New User Icon:** For users who joined less than 3 months ago.
+- **Translator Icon:** For users contributing as translators.
+- **Manager Icon:** For users serving as locale managers.
+- **Admin Icon:** For users holding administrative roles.
+
+# Technical Specification
+
+The badge system will be integrated into the existing Pontoon interface, with badges displayed on user profiles. Status icons will also be visible alongside user avatars in comments and suggestions. The system will track user activities and automatically award badges based on predefined criteria. Detailed implementation steps include:
+
+1. **Backend Development Criteria:**
+   - **Activity Tracking:** Track user activities such as translations submitted, translations reviewed, and promotions.
+   - **Award Logic:** Implement logic to award badges based on tracked activities.
+   - **Data Consideration:** Only new data will be considered for badges (i.e., the data will be tracked and counted towards the badges from the time the feature goes live).
+   - **Badge Levels:** Store badge levels and criteria.
+   - **Badge Replacement:** When a user reaches a new badge level, it will replace the older version of the badge.
+   - **Notifications:** Send notifications to users to communicate their badge achievements (e.g., advancing to a new badge level or obtaining a badge).
+   
+2. **Frontend Development:**
+   - **Badge Display:** Display badges on user profiles under an "Achievements" section. 
+     - Example: ![Achievements Section](0119/achievements-section.png)
+   - **Status Icons:** Show status icons universally in comments and suggestions.
+     - Example: ![Suggestions](0119/suggestions.png)
+     - Example: ![Comments](0119/comments.png)
+     - **Note:** The exact design of the status icons is not finalized yet. The circle on the user icon represents a placeholder.
+   - **New User Tag:** Within the user profile, display a "New User" tag if the new user criteria are met.
```

Comment:
I would drop this. If we show the status icons in avatars, it'll already appear in the profile page avatar.